deploy/helm/gateway-collector-config.yaml

```diff
@@ -156,6 +149,91 @@ processors:
           - cronjobs
           - pods
           - nodes
+      - address_attr: server.address
+        namespace_attr: k8s.namespace.name
+        workload_type_attr: sw.k8s.dst.workload.type
+        workload_name_attr: sw.k8s.dst.workload.name
+        workload_namespace_attr: sw.k8s.dst.workload.namespace
+        prefer_owner_for_pods: true
+        expected_types:
+          - services
+          - pods
+      - address_attr: client.address
+        namespace_attr: k8s.namespace.name
+        workload_type_attr: sw.k8s.src.workload.type
+        workload_name_attr: sw.k8s.src.workload.name
+        workload_namespace_attr: sw.k8s.src.workload.namespace
+        prefer_owner_for_pods: true
+        expected_types:
+          - services
+          - pods
+
+  transform/beyla-fqdn-attribute:
+    metric_statements:
+      - statements:
+          # The logic of what shapes of addresses are considered FQDNs should match 'transform/istio-metric-datapoints'.
+          # Also, Beyla prefers filling 'client.address' and 'server.address' attributes with OTEL Service name (or name + k8s namespace) instead of an actual FQDN, when possible. This makes their values useless for us because they would create fake FQDNs.
```

Comment:
This is unfortunate as it severely limits usefulness of Beyla for establishing HTTP/gRPC relationships between clients and servers. It was implemented that way intentionally. Though personally, I don't believe the reasoning there is correct.